--- a/text
+++ b/text
@@ -1 +1 @@
-Q: In terms of the use of water, Egypt uses almost 80% of its water for agriculture {{c1::in comparison}} (в сравнении) to domestic usage which {{c2::account for}} approximately 15% and 10% {{c3::respectively}} (соответственно).&nbsp;
+In Egypt, almost 80% of water is used for agriculture, {{c1::in comparison with}} domestic and industrial uses, which {{c2::account for}} about 15% and 10% {{c3::respectively}}.
