--- a/text
+++ b/text
@@ -1 +1 @@
-<div>Q: The proportion of energy from hydroelectric power, <b>…</b> remained constant in both years at a mere 5%.&nbsp;</div><div><br /></div><div>A: {{c1::on the other hand}} / {{c2::by contrast}} / {{c3::in contrast}}</div><div><br /></div>
+<div><b>Contrast linkers</b></div><div>1. {{c1::By contrast}}, hydroelectric power remained at 5% in both years.</div><div>2. {{c2::In contrast}}, hydroelectric power remained at 5% in both years.</div><div>3. {{c3::On the other hand}}, hydroelectric power remained at 5% in both years.</div>
